panel/src/components/Navigation/ButtonDisabled.vue

```diff
@@ -1,15 +1,16 @@
 <template>
-	<span
+	<button
 		:id="id"
 		:data-disabled="true"
 		:data-responsive="responsive"
 		:data-theme="theme"
 		:title="tooltip"
 		class="k-button"
+		aria-disabled="true"
```

Comment:
Shouldn't this just be disabled instead?

Reply:
No screenreaders will ignore it then. As I read it, it'd be a bad practice in terms of accessibility.

Reply:
Hm, so we basically need two types of disabled buttons, right? Those who should not be read, because it doesn't make sense (i.e. deactivated in dropdowns) and those who still need to convey a message (i.e. status)

Reply:
But why should they be visible but disabled for some people but completely non existent for screenreaders? Shouldn't they just as much get the option to focus it and get read out that there is this button/entry but disabled?

Reply:
Difficult question. There are quite a few folks who hate that we show disabled buttons. I've always belonged to the group who's advocating to show all available options, even if they are disabled. But I don't know how comfortable this is as a blind user. If I can quickly scan the list, I can ignore the disabled ones quite easily. But if I need to navigate through them one by one, it seems quite annoying.

Reply:
Not sure if more annoying or better than not being aware of its existence at all. Too much to assume for me not having any experience as screenreader user.

But the current situation needs to change. Don't think just about drop-down menu, but the status flag is the perfect example that even when disables still is there to communicate information. It must not continue that screenreaders are totally kept away from that information.

Reply:
Your last comment really convinced me. Then we should get this solution done for the 3.9 launch.